packages/engine/Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -3471,11 +3471,11 @@ describe(
       });
     });
 
-    it("Unload some cached tiles not required to meet SSE using maximumMemoryUsage", function () {
+    it("Unload some cached tiles not required to meet SSE using cacheBytes", function () {
```

Comment:
Since it was deprecated, can we leave the old tests specifically for `maximumMemoryUsage`, and add new ones to cover `cacheBytes`?